.gitignore

```diff
@@ -36,3 +36,4 @@ tags
 # IDE support files
 .idea
 
+.vscode/*
```

Comment:
:astonished: Are you using vscode for Boulder & friends regularly?

Reply:
...yes, visual code studio is great :heart:.